src/naming/Naming_AST.mli

```diff
@@ -3,3 +3,4 @@
  * specific to a language.
  *)
 val resolve : Lang.t -> AST_generic.program -> unit
+val is_js_angular_decorator : string -> bool
```

Comment:
This is ugly. Ideally those functions would be in another file, or there will be a comment in the .mli explaining
why the hell we need to do something JS and even Angular specific in what is supposed to be language agnostic.